config/settings.yml

```diff
@@ -1306,6 +1306,7 @@
         :ems_metrics_collector_worker_openstack_network: {}
         :ems_metrics_collector_worker_redhat: {}
         :ems_metrics_collector_worker_vmware: {}
+        :ems_metrics_openstack_default_service: "auto"
```

Comment:
Not quite - auto defaults to the current code path, where CF will attempt to connect to Gnocchi; if it can't, it'll then try Ceilometer.  That means the default behavior is the same as the current behavior.